spaces/farm/skin.scss

```diff
@@ -0,0 +1,3 @@
+.name{
```

Comment:
The css selector should be farm instead of name

Reply:
sure